pkg/domain/infra/abi/cp.go

```diff
@@ -0,0 +1,435 @@
+// +build ABISupport
```

Comment:
Can we remove this? That's in a race with https://github.com/containers/libpod/pull/5863#issuecomment-616710773 and can lead to failing gating tests with #5863. Swagger doesn't support build tags and goes south in certain cases.